### ISSUE TITLE
unknown answer: "exception":"Invalid_argument

### ISSUE THREAD

**Issue by cpitclaudel** [ocaml/merlin#1010]
I just ran across the following issue when trying to generate a match statement on a record:

```
cond: unknown answer: "exception":"Invalid_argument(\"get_every_pattern: [{\\\"filename\\\":\\\"cpp.ml\\\",\\\"start\\\":{\\\"line\\\":131,\\\"col\\\":8},\\\"end\\\":{\\\"line\\\":131,\\\"col\\\":51},\\\"ghost\\\":false,\\\"attrs\\\":[],\\\"kind\\\":\\\"value_binding\\\",\\\"children\\\":[{\\\"filename\\\":\\\"cpp.ml\\\",\\\"start\\\":{\\\"line\\\":131,\\\"col\\\":12},\\\"end\\\":{\\\"line\\\":131,\\\"col\\\":13},\\\"ghost\\\":false,\\\"attrs\\\":[],\\\"kind\\\":\\\"pattern (cpp.ml[131,3286+12]..cpp.ml[131,3286+13])\\\\n  Tpat_var \\\\\\\"b/216628\\\\\\\"\\\\n\\\",\\\"children\\\":[]},{\\\"filename\\\":\\\"cpp.ml\\\",\\\"start\\\":{\\\"line\\\":131,\\\"col\\\":16},\\\"end\\\":{\\\"line\\\":131,\\\"col\\\":51},\\\"ghost\\\":false,\\\"attrs\\\":[],\\\"kind\\\":\\\"expression\\\",\\\"children\\\":[{\\\"filename\\\":\\\"cpp.ml\\\",\\\"start\\\":{\\\"line\\\":131,\\\"col\\\":16},\\\"end\\\":{\\\"line\\\":131,\\\"col\\\":46},\\\"ghost\\\":false,\\\"attrs\\\":[],\\\"kind\\\":\\\"expression\\\",\\\"children\\\":[]},{\\\"filename\\\":\\\"cpp.ml\\\",\\\"start\\\":{\\\"line\\\":131,\\\"col\\\":47},\\\"end\\\":{\\\"line\\\":131,\\\"col\\\":51},\\\"ghost\\\":false,\\\"attrs\\\":[],\\\"kind\\\":\\\"expression\\\",\\\"children\\\":[]}]}]}]\")
Raised at file \"stdlib.ml\", line 34, characters 20-45
Called from file \"src/analysis/destruct.ml\", line 426, characters 34-59
Called from file \"src/ocaml/utils/misc.ml\", line 45, characters 8-15
Re-raised at file \"src/ocaml/utils/misc.ml\", line 57, characters 10-24
Called from file \"src/utils/std.ml\", line 679, characters 8-12
Re-raised at file \"src/utils/std.ml\", line 681, characters 30-39
Called from file \"src/utils/local_store.ml\", line 29, characters 8-12
Re-raised at file \"src/utils/local_store.ml\", line 37, characters 4-15
Called from file \"src/kernel/mocaml.ml\", line 40, characters 8-38
Re-raised at file \"src/kernel/mocaml.ml\", line 48, characters 4-15
Called from file \"src/frontend/new/new_commands.ml\", line 65, characters 15-53
Called from file \"src/utils/std.ml\", line 679, characters 8-12
Re-raised at file \"src/utils/std.ml\", line 681, characters 30-39
Called from file \"src/ocaml/utils/misc.ml\", line 45, characters 8-15
Re-raised at file \"src/ocaml/utils/misc.ml\", line 57, characters 10-24
Called from file \"src/frontend/new/new_merlin.ml\", line 99, characters 18-54
"
```

Is this enough data to track the issue?

**Comment by trefis** [comment]
> Is this enough data to track the issue?

Should be.
We're a bit short on time to investigate this at the moment. It should get better at the beginning of 2020.

**Comment by voodoos** [comment]
Fixed by #1178
